CHANGELOG-3.1.md

```diff
@@ -1,4 +1,16 @@
-## [v3.1.12](https://github.com/coreos/etcd/releases/tag/v3.1.12) (TBD)
+
+## [v3.1.12](https://github.com/coreos/etcd/releases/tag/v3.1.13) (TBD)
+
+See [code changes](https://github.com/coreos/etcd/compare/v3.1.12...v3.1.13) and [v3.1 upgrade guide](https://github.com/coreos/etcd/blob/master/Documentation/upgrades/upgrade_3_1.md) for any breaking changes.
+
+### Improved
+
+- Adjust [election timeout on server restart](https://github.com/coreos/etcd/pull/9364) to reduce [disruptive rejoining servers](https://github.com/coreos/etcd/issues/9333).
+  - Previously, etcd fast-forwards election ticks on server start, with only one tick left for leader election. This is to speed up start phase, without having to wait until all election ticks elapse. Advancing election ticks is useful for cross datacenter deployments with larger election timeouts. However, it was affecting cluster availability if the last tick elapses before leader contacts the restarted node.
+  - Now, when etcd restarts, it does not fast-forward election ticks.
```

Comment:
> Now, when etcd restarts, it adjusts election ticks with more than one tick left, thus more time for leader to prevent disruptive restart.

https://github.com/coreos/etcd/pull/9415 is merged, where it adjust election ticks on restart.

Reply:
Updated CHANGELOGs to use reference #9415